ksqldb-execution/src/main/java/io/confluent/ksql/execution/plan/KStreamHolder.java

```diff
@@ -25,21 +25,21 @@
 public final class KStreamHolder<K> {
 
   private final KStream<K, GenericRow> stream;
-  private final KeySerdeFactory<K> keySerdeFactory;
+  private final ExecutionKeyFactory<K> executionKeyFactory;
   private final LogicalSchema schema;
 
   public KStreamHolder(
       final KStream<K, GenericRow> stream,
       final LogicalSchema schema,
-      final KeySerdeFactory<K> keySerdeFactory
+      final ExecutionKeyFactory<K> executionKeyFactory
   ) {
     this.stream = Objects.requireNonNull(stream, "stream");
-    this.keySerdeFactory = Objects.requireNonNull(keySerdeFactory, "keySerdeFactory");
+    this.executionKeyFactory = Objects.requireNonNull(executionKeyFactory, "keySerdeFactory");
     this.schema = Objects.requireNonNull(schema, "schema");
   }
 
-  public KeySerdeFactory<K> getKeySerdeFactory() {
-    return keySerdeFactory;
+  public ExecutionKeyFactory<K> getKeySerdeFactory() {
```

Comment:
Rename to `getKeyFactory()` or `getExecutionKeyFactory()`?

Reply:
It doesn't look like changes were made even though this comment was resolved?

Reply:
ah oops! I had made this change somewhere else and just assumed that was the place this was pointing at. I'll update it